src/runtime_src/tools/scripts/pkg_hw_platform.sh

```diff
@@ -460,6 +460,8 @@ initXsaBinEnvAndVars()
          fwManagement="${XILINX_XRT}/share/fw/mgmt.bin"
       elif [ "${CardMgmtControllerFamily}" == "CMC-Gen1" ]; then
          fwManagement="${XILINX_XRT}/share/fw/cmc.bin"
+      elif [ "${CardMgmtControllerFamily}" == "CMC-NO-SC-GEN1" ]; then
```

Comment:
Hello Raj,
In a previous e-mail we had discussed going with a name similar to: "CMC-NoSC-Gen1"
The key here is to NOT have a dash '-' between "NO" and "SC" and I was wondering why the instance of the dash.

In addition, all caps are being used, which is different from the current camel case convention.

Reply:
Raj, Thank you for making this change.